DocAI Incubator Tools/best-practices/Document_AI_Parser_Result_Merger.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<p>Change the parameter names to be displayed in code syntax</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>